kingpin/actors/rightscale/api.py

```diff
@@ -227,7 +227,7 @@ def find_right_script(self, name):
     @concurrent.run_on_executor
     @sync_retry(**settings.RETRYING_SETTINGS)
     @utils.exception_logger
-    def find_by_name_and_keys(self, collection, name, exact=True, **kwargs):
+    def find_by_name_and_keys(self, collection, exact=True, **kwargs):
```

Comment:
Actually its not -- name still works just fine, I just removed the
'requirement' that you supply it. There are cases where you want to search
on something else (like a resource_uid). However, I may nee to sanity check
our code and make sure we're always passing in `name=<myname>` rather than
just `<myname>`.

Matt Wise
Sr. Systems Architect
Nextdoor.com

On Wed, Oct 21, 2015 at 2:05 PM, Mikhail Simin <notifications@github.com>
wrote:

> In kingpin/actors/rightscale/api.py
> <https://github.com/Nextdoor/kingpin/pull/289#discussion_r42682161>:
>
> > @@ -227,7 +227,7 @@ def find_right_script(self, name):
> >      @concurrent.run_on_executor
> >      @sync_retry(**settings.RETRYING_SETTINGS)
> >      @utils.exception_logger
> > -    def find_by_name_and_keys(self, collection, name, exact=True, **kwargs):
> > +    def find_by_name_and_keys(self, collection, exact=True, **kwargs):
>
> Why are you removing name? I have code that's leveraging that
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Nextdoor/kingpin/pull/289/files#r42682161>.
>